### PR TITLE
Create cms-tools-ana-2011.xml

### PR DESCRIPTION
**Pull request by AnxhelaDani**
First draft of the needed records for the analysis example-2011 and its software for https://github.com/cernopendata/opendata.cern.ch/issues/905#issuecomment-191828967

### CODE REVIEW

File: invenio_opendata/testsuite/data/cms/cms-tools-ana-2011.xml [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+      <subfield code="b">CERN Open Data Portal</subfield>
+      <subfield code="c">2016</subfield>
+    </datafield>
+    <datafield tag="264" ind1=" " ind2="0">
+      <subfield code="c">2016</subfield>
+    </datafield>
+    <datafield tag="516" ind1=" " ind2=" ">
+      <subfield code="a">Use this with the following datasets:</subfield>
+      <subfield code="w">FIXME</subfield>
+      <subfield code="w">FIXME</subfield>
```
Comment (tiborsimko) [inline invenio_opendata/testsuite/data/cms/cms-tools-ana-2011.xml]: Which datasets to use here?

File: invenio_opendata/testsuite/data/cms/cms-tools-ana-2011.xml [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    </datafield>
+    <datafield tag="520" ind1=" " ind2=" ">
+      <subfield code="a">Software to produce the intermediate data files, derived from the primary datasets, for a simple analysis on Z decays to two leptons and ZZ decays to four leptons, of CMS 2011 open data. </subfield>
+    </datafield>
+    <datafield tag="538" ind1=" " ind2=" ">
+      <subfield code="a">Software release: CMSSW_5_3_32</subfield>
+      <subfield code="i">Use this code with the CMS Open Data VM environment for 2011 open data</subfield>
+      <subfield code="w">FIXME:-252-</subfield>
```
Comment (tiborsimko) [inline invenio_opendata/testsuite/data/cms/cms-tools-ana-2011.xml]: FIXME:-252- ?

File: invenio_opendata/testsuite/data/cms/cms-tools-ana-2011.xml [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+      <subfield code="w">1001</subfield>
+    </datafield>
+    <datafield tag="693" ind1=" " ind2=" ">
+      <subfield code="a">CERN-LHC</subfield>
+      <subfield code="e">CMS</subfield>
+    </datafield>
+    <datafield tag="787" ind1=" " ind2=" ">
+      <subfield code="a">The files created are:</subfield>
+      <subfield code="w">FIXME</subfield>
+      <subfield code="w">FIXME</subfield>
```
Comment (tiborsimko) [inline invenio_opendata/testsuite/data/cms/cms-tools-ana-2011.xml]: Which values to use here?

File: invenio_opendata/testsuite/data/cms/cms-tools-ana-2011.xml [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+      <subfield code="b">CERN Open Data Portal</subfield>
+      <subfield code="c">2016</subfield>
+    </datafield>
+    <datafield tag="264" ind1=" " ind2="0">
+      <subfield code="c">2016</subfield>
+    </datafield>
+    <datafield tag="516" ind1=" " ind2=" ">
+      <subfield code="a">Use this with the following datasets:</subfield>
+      <subfield code="w">FIXME</subfield>
+      <subfield code="w">FIXME</subfield>
```
Comment (tiborsimko) [inline invenio_opendata/testsuite/data/cms/cms-tools-ana-2011.xml]: Which values to use here?

File: invenio_opendata/testsuite/data/cms/cms-tools-ana-2011.xml [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+    </datafield>
+    <datafield tag="520" ind1=" " ind2=" ">
+      <subfield code="a">Example code for a simple analysis on Z decays to two leptons and ZZ decays to four leptons, of CMS 2011 open data.</subfield>
+    </datafield>
+    <datafield tag="538" ind1=" " ind2=" ">
+      <subfield code="a">Software release: CMSSW_5_3_32</subfield>
+      <subfield code="i">Use this code with the CMS Open Data VM environment for 2011 open data</subfield>
+      <subfield code="w">FIXME:-252-</subfield>
```
Comment (tiborsimko) [inline invenio_opendata/testsuite/data/cms/cms-tools-ana-2011.xml]: FIXME:-252- ?

File: invenio_opendata/testsuite/data/cms/cms-tools-ana-2011.xml [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+      <li>χ2 /ndo f &lt; 10 for the global muon fit;</li>
+      <li>Transverse impact parameter w.r.t. the beam spot &lt; 0.02 cm;</li>
+      <li>Symmetrised impact parameter w.r.t. the beam sport &lt; 4 cm;</li>
+      <li>Relative Muon Isolation (RMI) &lt; 0.15, with RMI = (ChargedHadronIso + NeutralHadronIso + PhotonIso)/pT(µ)</li></ul>
+      <br/>Each component is computed from Isodeposits in a cone of 0.3 around the muon direction. </p> ]]></subfield>
+    </datafield>
+    <datafield tag="581" ind1=" " ind2=" ">
+      <subfield code="a">See the run instructions in</subfield>
+      <subfield code="u">FIXME</subfield>
+      <subfield code="y">FIXME</subfield>
```
Comment (tiborsimko) [inline invenio_opendata/testsuite/data/cms/cms-tools-ana-2011.xml]: Which values to use here?

File: invenio_opendata/testsuite/data/cms/cms-tools-ana-2011.xml [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+      <subfield code="a">The input files contain only the validated runs for primary datasets of 2011. For user validation of the analysis step, compare the result to the expected output file.</subfield>
+      <subfield code="w">1001</subfield>
+    </datafield>
+    <datafield tag="693" ind1=" " ind2=" ">
+      <subfield code="a">CERN-LHC</subfield>
+      <subfield code="e">CMS</subfield>
+    </datafield>
+    <datafield tag="787" ind1=" " ind2=" ">
+      <subfield code="a">The expected output is a histogram file such as</subfield>
+      <subfield code="o">FIXME</subfield>
```
Comment (tiborsimko) [inline invenio_opendata/testsuite/data/cms/cms-tools-ana-2011.xml]: Which value to use here?

File: invenio_opendata/testsuite/data/cms/cms-tools-ana-2011.xml [hunk 8]
```diff
@@ -0,0 +1,178 @@
+<collection>
+  <record>
+    <controlfield tag="001">FIXME</controlfield>
+    <datafield tag="024" ind1="7" ind2=" ">
+      <subfield code="2">DOI</subfield>
+      <subfield code="a">10.7483/OPENDATA.CMS.ETJK.JKMB</subfield>
+    </datafield>
+    <datafield tag="100" ind1=" " ind2=" ">
+      <subfield code="a">Rodriguez Marrero, Ana</subfield>
```
Comment (tiborsimko) [inline invenio_opendata/testsuite/data/cms/cms-tools-ana-2011.xml]: Shouldn't Kati be a co-author (in 700)?  We are pointing to her place...

File: invenio_opendata/testsuite/data/cms/cms-tools-ana-2011.xml (hunk 1 above)
Comment (katilp) [inline invenio_opendata/testsuite/data/cms/cms-tools-ana-2011.xml]: http://opendatadev.cern.ch/record/17 and http://opendatadev.cern.ch/record/16

File: invenio_opendata/testsuite/data/cms/cms-tools-ana-2011.xml (hunk 3 above)
Comment (katilp) [inline invenio_opendata/testsuite/data/cms/cms-tools-ana-2011.xml]: If this refers to the output files (i.e. the pattuples), they are the one that will be uploaded to eospublic by Aida and Jesus

File: invenio_opendata/testsuite/data/cms/cms-tools-ana-2011.xml (hunk 4 above)
Comment (katilp) [inline invenio_opendata/testsuite/data/cms/cms-tools-ana-2011.xml]: the records is #959

File: invenio_opendata/testsuite/data/cms/cms-tools-ana-2011.xml (hunk 6 above)
Comment (katilp) [inline invenio_opendata/testsuite/data/cms/cms-tools-ana-2011.xml]: https://github.com/katilp/OutreachExercise2011/blob/master/README.md
(or https://github.com/katilp/OutreachExercise2011)

**Comment by AnxhelaDani** [comment]
@tiborsimko  changes done, ready for merge.

**Comment by tiborsimko** [comment]
Thanks, squashed, fixed CDATA formatting, renamed file for consistency, and merged as:

- ff962b2a3cdd60efd9aa08bb063710a8177eadcd CMS: new file name for Ana 2011 file
- 55194f5ece5626cf0427f48dbe295b3fd3a920ff CMS: fix CDATA in Ana 2011 record
- f599f1728f9336e79116b1f97e00e0c98f9973e5 CMS: new `cms-tools-ana-2011.xml`

**Pull request closed without merging**